contracts/MetaverseBaseNFT.sol

```diff
@@ -156,11 +152,6 @@ contract MetaverseBaseNFT is
         return super.tokenURI(tokenId);
     }
 
-    function totalSupply() public view returns (uint256) {
```

Comment:
I think we still need this, it's shown on the frontend widget when minting. Is it not posssible with ERC721A to really export that?

Reply:
@caffeinum ERC721A contract has `totalSupply()` function so I think we don't need to override it. It's good enough to go.

https://github.com/chiru-labs/ERC721A/blob/d6d761bc2678de9b179fcc812f26891bf1871fb5/contracts/ERC721A.sol#L69